Index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Style.css">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+</head>
+<body>
+
+ <header class="top-bar">
+  <a href="#" id="logolink"><h2 class="logoname">My Logo</h2></a>
+  <nav >
+    <img src="images\Union.png" alt="logo">
+  </nav>
+</header>
+ <section class="section-data">
+    <h1 class="myname">I’m Surya
+      Glad to see you!</h1>
+    <p class="namedetail">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div>
+    <h3 class="connect">LET’S CONNECT</h3>
+      <ul class="socialmedia">
+        <li><a href="#"><img class="twitter" src="images\twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img class="dot" src="images\dot.png" alt="dot"></a></li>
+        <li><a href="#"><img class="i" src="images\i.png" alt="i"></a></li>
+        <li><a href="#"><img class="n" src="images\n.png" alt="n"></a></li>
+        <li><a href="#"><img class="m" src="images\M.png" alt="m"></a></li>
+        <li><a href="#"><img class="g" src="images\g.png" alt="g"></a></li>
+        <li><a href="#"><img class="v" src="images\v.png" alt="v"></a></li>
+      </ul>
```

Comment:
- [ ] Please kindly ensure that the margin of these images is close to what we have in the figma design; and please don't forget to display the it as flex as suggested earlier. 👍 The flex CSS shorthand property sets how a flex item will grow or shrink to fit the space available in its flex container. ℹ️ 

## Your Design
![DSC_7491](https://user-images.githubusercontent.com/103386183/219430724-6c9ca7be-33a9-4bdf-aaa9-eb48ad3ccada.JPG)